json_rails_logger.gemspec

```diff
@@ -1,16 +1,25 @@
 # frozen_string_literal: true
 
-require './lib/json_rails_logger/version'
+require 'json'
+pkg = JSON.parse(File.read('./package.json'))
 
 Gem::Specification.new do |s|
   s.name        = 'json_rails_logger'
-  s.version     = JsonRailsLogger::VERSION
+  s.version     = pkg['version']
   s.date        = '2021-01-06'
   s.summary     = 'JSON Rails Logger'
   s.description = 'A custom rails logger that outputs JSON instead of raw text'
   s.authors     = ['Bogdan-Adrian Marc']
   s.email       = 'bogdan.marc@epimorphics.com'
-  s.files       = ['./lib/json_rails_logger.rb']
+  s.files       = [
```

Comment:
A better solution is #27

Reply:
yes.  are you expecting #27 to get merged in here?

Reply:
Yes, but it hasn't been approved yet

Reply:
I'd rather not create a dependency on #27. Get this approved as it and #27 applied subsequently.

Reply:
Has been merged now (the approval was in my email backlog)